src/temperature/schemas.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+from src.city.schemas import City
+
+
+class BaseTemperature(BaseModel):
+    ...
+
+
+class Temperature(BaseTemperature):
+    id: int
+    date_time: datetime
+    temperature: str
```

Comment:
The `temperature` field is defined as a `str`. If temperatures are stored as numeric values, consider using a `float` type instead. This will allow for more accurate representation and validation of temperature data.